### ISSUE TITLE
Tooltips for icons

### ISSUE THREAD

**Issue by JasMich** [PapyElGringo/material-shell#168]
All the icons in the two panels are missing explaining tooltips. This is a huge accessibility issue and decreases usability for disables people or new users not yet known to Material Shell.

**Comment by PapyElGringo** [comment]
I understand that it's could be a benefit but do we agree that there isn't any tooltip either in the default gnome panel?

**Comment by JasMich** [comment]
But there are labels for screen readers. Specially the "switch layout" button would need a label as the symbols my not be that explaining even for non-disabled people.

**Comment by dwrolvink** [comment]
Switch layout now has labels for each layout option. 
![image](https://user-images.githubusercontent.com/30291690/97780643-28435100-1b86-11eb-9536-4961c2382c41.png)

Clicking on any of the icons will make it also apparent what the function of each icon is. 

I don't think you need a tool tip for these either, as they are extremely self-explanatory:
![image](https://user-images.githubusercontent.com/30291690/97780580-cd115e80-1b85-11eb-949d-b250234bb881.png)
![image](https://user-images.githubusercontent.com/30291690/97780586-d3073f80-1b85-11eb-989f-8cf1a91d3885.png)

Not much left then.
